common-tooling/action.yml

```diff
@@ -262,6 +264,32 @@ runs:
       cache-dependency-path: ${{ inputs.python-cache-dependency-path }}
       update-environment: ${{ inputs.python-update-environment }}
       allow-prereleases: ${{ inputs.python-allow-prereleases }}
+
+  - name: Install Python3 (ubuntu-arm install)
+    uses: deadsnakes/action@v3.1.0
+    # actions/setup-python doesn't yet support ARM, see https://github.com/actions/setup-python/issues/705#issuecomment-1756948951
+    if: ${{ contains(runner.arch, 'arm') && contains(runner.name, 'ubuntu') && inputs.python-enabled == 'true' && ( env.python_missing == 'true' || inputs.overwrite == 'true' ) }}
```

Comment:
I don't think this is executed on arm runners since the if can never be true due to `contains(runner.name, 'ubuntu'`) afaik self-hosted runners should never contain `ubuntu` in their name, therefore skipped.
I also didn't find it called in the logs.